inductiva/_cli/cmd_tasks/kill.py

```diff
@@ -1,13 +1,21 @@
 """Kills a tasks by id via CLI."""
 
 import inductiva
+from inductiva import constants
 
 
 def kill_task(args):
```

Comment:
The following can be addressed in another PR, but should we permit killing also all running tasks? Wdyt?

Reply:
maybe at some point.
Open a task for this.

This is why that document is important. It should be kept up-to-date so that we can check for consistency